tests/integration/challenges/memory/test_memory_challenge_c.py

```diff
@@ -0,0 +1,92 @@
+import pytest
```

Comment:
Great !
Could you update the mkdocs.yml
```
- Memory:
    - Introduction: challenges/memory/introduction.md
    - Memory Challenge A: challenges/memory/challenge_a.md
    - Memory Challenge B: challenges/memory/challenge_b.md
    - Memory Challenge C: challenges/memory/challenge_c.md
```

And then put a description of the challenges in memory/challenge_c.md (you can copy memory/challenge_b.md and modify it)

Reply:
Done!